docs/source/modules/simcloud.rst

```diff
@@ -0,0 +1,65 @@
+Cloud Scheme (SimCloud)
+=======================
+
+Summary
+-------
+Isca provides a simple diagnostic cloud scheme, SimCloud, originally developed and described in [Liu2021]_. The model diagnoses clouds in three different ways: (1) large-scale clouds from the relative humidity; (2) low-level marine stratus clouds from a function of local inversion strength; and (3) a "freeze-dry" adjustment based on a simple function of specific humidity is also available to reduce an excessive cloud bias in polar regions. Cloud properties such as the effective radius of cloud droplet and cloud liquid water content are specified as simple functions of temperature, and parameters of the model may be tuned individually to suit requirements.
+
+The cloud scheme may be found in the directory ``src/atmos_param/cloud_simple``, with tuneable parameters found therein.
+
+A demonstration test case is provided within Isca's test suite of experiments. The scheme is provided on an as-in basis: parameters have been tuned for an Earth-like demonstration so any deviations from such a setup or modifications to the parameterisation are at your own risk. The model should be used with the SOCRATES radiation scheme.
```

Comment:
Can you provide the path and name please?

Reply:
Can I also suggest that you comment that there are 2 cloud schemes in the model and that this is the one we recommend people use?